src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/CoreTests_ComprehensiveRegionProperties.sarif

```diff
@@ -11,6 +11,30 @@
         "sarifLoggerVersion": "1.5.22.0",
         "language": "en-US"
       },
+      "invocations": [
```

Comment:
>invocations [](start = 7, length = 11)

For the sake of brevity, I suggest we not include optional elements that aren't related to the test...

Oh... is your idea to test that (for example), when _only_ the "populate comprehensive region properties" flag is set, that it does _not_ also flatten the notifications? That makes sense. Ok. Stream of consciousness comment withdrawn.